TAF_EasyRest/TestFramework/Pages/AdminPanelModeratorsPage.cs

```diff
@@ -6,41 +6,52 @@ namespace TestFramework.Pages
     public class AdminPanelModeratorsPage : BasePage
     {
 
-        public NavigationMenuPageComponent NavigationMenuPageComponent { get; }
-        public AdminPanelPageComponent AdminPanelPageComponent { get; }
+        public AdminPanelPageComponent AdminLeftsideMenu { get; }
+        public NavigationMenuPageComponent NavigationMenu { get; }
+        public UserMenuHeaderButtonPageComponent UserButton { get; }
 
 
+        public AdminPanelModeratorsPage(IWebDriver driver) : base(driver)
+        {
+            AdminLeftsideMenu = new AdminPanelPageComponent(driver);
+            NavigationMenu = new NavigationMenuPageComponent(driver);
+            UserButton = new UserMenuHeaderButtonPageComponent(driver);
+        }
+
+        private IReadOnlyCollection<IWebElement> _moderatorsList => driver.FindElements(By.XPath("//tbody/tr"));
+
+
+        #region Elements
+
         private IWebElement _addModeratorButton => driver.FindElement(By.XPath("//*[@id = 'root']/div/main/a"));
 
-        // Corresponding entry button. It's better to find 1-st row instead of certain user name. Like //tbody/tr[1]
-        private IWebElement _padlockButton => driver.FindElement(By.XPath("//th[text()='Peter Moderator']/following-sibling::td/button"));
-        private IWebElement _statusRecord => driver.FindElement(By.XPath("//th[text()='Peter Moderator']/following-sibling::td/p"));
-        private IWebElement _padlockButtonForCertainUser(string username) => driver.FindElement(By.XPath($"//th[text()='{username}']/following-sibling::td/button"));
+        #endregion
 
 
+        #region Methods
 
-        public AdminPanelModeratorsPage(IWebDriver driver) : base(driver)
+        // Find first row of the moderators list. Find button and status for it. Click button. Check if the status has changed
+        public bool ChangeModeratorStatus()
         {
-            AdminPanelPageComponent = new AdminPanelPageComponent(driver);
-            NavigationMenuPageComponent = new NavigationMenuPageComponent(driver);
-        }
+            var firstModeratorFromList = _moderatorsList.ElementAt(0);
+            var padlockButton = firstModeratorFromList.FindElement(By.XPath("./td/button"));
+            var statusRecord = firstModeratorFromList.FindElement(By.XPath("./td/p"));
```

Comment:
I don't understand what `statusRecord` and `startStatus` mean in the context of your page object. Please explain.

Reply:
I removed these variables. In the context of my page object, I need only a label, that shows the current status of the moderator.